src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -595,9 +595,9 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
         }))
```

Comment:
This still displays the static data when trying to add to a Req List:

```
requisitionListData: {
        list: [
          { name: 'Vancouver' }, { name: 'HQ' }, { name: 'Chicago' }, { name: 'New York' }, { name: 'San Francisco' }, { name: 'Toronto' }, { name: 'Lviv' }, { name: 'Denver' }, { name: 'San Diego' },
        ],
      },requisitionListData: {
        list: [
          { name: 'Vancouver' }, { name: 'HQ' }, { name: 'Chicago' }, { name: 'New York' }, { name: 'San Francisco' }, { name: 'Toronto' }, { name: 'Lviv' }, { name: 'Denver' }, { name: 'San Diego' },
        ],
      },requisitionListData: {
        list: [
          { name: 'Vancouver' }, { name: 'HQ' }, { name: 'Chicago' }, { name: 'New York' }, { name: 'San Francisco' }, { name: 'Toronto' }, { name: 'Lviv' }, { name: 'Denver' }, { name: 'San Diego' },
        ],
      },

Reply:
@shaunmaharaj Good catch, I accidentally deleted this changes when resolving conflicts